resources/views/tasks/show.blade.php

```diff
@@ -1,14 +1,111 @@
 @extends('layouts.layout')
 
 @section('content')
-<div class="container" id="task">
-    <task-view
-        process-uid="{{$process->uid}}" instance-uid="{{$instance->uid}}" token-uid="{{$token->uid}}" form-uid="{{$token->definition['formRef']}}"
-        :data="{{json_encode($data)}}" token-created="{{$token->delegate_date}}" token-completed="{{$token->finish_date}}"
-        user-uid="{{$token->user->uid}}" user-name="{{$token->user->fullname}}" user-avatar="{{$token->user->avatar}}"></task-view>
+<div>
+    <div class="nav row taskNav">
+        <div class="col-3 align-self-center">
+            <a class="nav-link active text-light p-0 ml-3" href="#"><i class="fas fa-long-arrow-alt-left fa-lg mr-2"></i>BACK TO REQUEST DETAILS</a>
+        </div>
+        <div class="col-3 align-self-center text-right">
+            <a class="nav-link text-light p-0" href="#"><h4 class="m-0">Task: Approve Leave of Absence</h4></a>
+        </div>
+        <div class="col align-self-center">
+            <a class="nav-link p-0" href="#" ><span class="pill badge-light p-1 pl-2 pr-2"><i class="fas fa-circle text-primary mr-2"></i><span>Pending</span></a>
+        </div>
+    </div>
+    <br>
+    <div class="d-flex container">
+        <br>
+        <div class="col-9">
+            <br>
+            <div class="container p-5">
+                <h3>Request</h3>
+                <div id="request">
+                </div>
+            </div>
+            <br>
+            <br>
+            <br>
+            <div class="row">
```

Comment:
Remove comments. This is an enterprise feature.

Reply:
removed